### PR TITLE
Support Kensu-Py 2.0.0, download spark jar

### PR DESCRIPTION
**Pull request by leszekgruchala**

### CODE REVIEW

File: kensu-spark-example/script_with_kensu.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 #Add the path to the .jar to the SparkSession
 spark = SparkSession.builder.appName("Example")\
-    .config("spark.driver.extraClassPath", "kensu-dam-spark-collector-0.17.2_spark-3.0.1.jar")\
+    .config("spark.driver.extraClassPath", "kensu-spark-collector-1.0.0-rc466_spark-3.0.1.jar")\
```
Comment (vidma) [inline kensu-spark-example/script_with_kensu.py]: download from public http server instead?

**Review by vidma — COMMENTED** [review]
ok otherwise

File: kensu-spark-example/script_with_kensu.py (hunk 1 above)
Comment (leszekgruchala) [reply kensu-spark-example/script_with_kensu.py]: Very good idea

**Comment by leszekgruchala** [comment]
@vidma Answered your comments, can you please make the code changes from now on across the integration projects?

**Pull request merged**